### ISSUE TITLE
Dark Color Showed with lack of PC info

### ISSUE THREAD

**Issue by YakoBen** [rcmaehl/WhyNotWin11#393]
**Describe the bug**
When I opened the app, the color is black and just showing the status, "OK" buttons.

**To Reproduce**
1. Right-click on your mouse.
2. Hover down to "Personalize"
3. After Theme settings opens in the control panel, click on "Colors" on the right sections.
4. Go to "Choose your color", and try to get back to your theme default settings or change the option to "Custom" and try to change Dark to Light or vise versa, Then it will work. "This is what happened with me".

**Expected behavior**
The color will get back to white and you will see everything, and the app settings will get back to default.
![Issue-whynotwin11-fixed](https://user-images.githubusercontent.com/86914780/124384188-9f098e80-dcbf-11eb-8a24-064c7f7be60b.png)

**Screenshots**
![Issue-Whynotwin11](https://user-images.githubusercontent.com/86914780/124383985-e04d6e80-dcbe-11eb-976b-8b56c2254673.png)

**Desktop (please complete the following information):**
 - OS: Win 10

**Comment by JohnLGalt** [comment]
Same on 11, too.

**Comment by micwoj92** [comment]
@YakoBen 
Could you try https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip ?

**Comment by svenrat** [comment]
Same problem here, but with a solution. If there is no registry entry for "AppsUseLightTheme" (I have Win8.1), then I get this behavior. So I changed the code.

At the end of "WhyNotWin11.au3" I add the following lines:

```
Func _CheckAppsUseLightTheme()
	Local $sUseLightTheme = RegRead("HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Themes\Personalize", "AppsUseLightTheme")
	if @error then
		$sUseLightTheme = "1"
	EndIf
	Return Int($sUseLightTheme)
EndFunc	;==>_AppsUseLightTheme
```

and then I changed the following lines to:

```
329:  Local $sCheck = _CheckAppsUseLightTheme()
893:  Case $sSysWin = 16777215 And Not _CheckAppsUseLightTheme()
```

Then it works for me without changing some Theme stuff :) .

**Comment by micwoj92** [comment]
Can confirm it fixes issue on 8.1
![image](https://user-images.githubusercontent.com/45581170/124506022-15df7e00-ddcb-11eb-9f78-7dccbb81c308.png)
